src/axom/quest/SignedDistance.hpp

```diff
@@ -70,14 +70,16 @@ class SignedDistance
    * \param [in] surfaceMesh user-supplied surface mesh.
    * \param [in] isWatertight indicates if the surface mesh is closed.
    * \param [in] maxObjects max number of objects for spatial decomposition.
-   * \param [in] maxLevels max levels for spatial decomposition (optional).
+   * \param [in] maxLevels max levels for spatial decomposition.
+   * \param [in] computeSign indicates if distance queries should compute signs.
    * \note Default maxLevels is 5 if not specified.
    * \pre surfaceMesh != nullptr
    */
   SignedDistance( const mint::Mesh* surfaceMesh,
                   bool isWatertight,
                   int maxObjects,
-                  int maxLevels   );
+                  int maxLevels,
+                  bool computeSign );
```

Comment:
Let's make `computeSign` optional here and always default to `true` since the class is called `SignedDistance`...

Reply:
ok